src/rust/catnip/runtime/mod.rs

```diff
@@ -402,103 +400,23 @@ impl NetworkRuntime for SharedDPDKRuntime {
 
     fn transmit<P: PacketBuf>(&mut self, mut pkt: P) -> Result<(), Fail> {
         timer!("catnip::runtime::transmit");
+        // Grab the packet and copy it if necessary. In general, this copy will happen for small packets without ////..
+        // payloads because we allocate actual data-carrying application buffers from the DPDK pool.
+        let outgoing_pkt: DemiBuffer = match pkt.take_body().unwrap() {
```

Comment:
This function returns a `Result`. Why should we `unwrap()` or `expect()`, if we can log an error message and fail instead?